rules-tests/PostRector/AdditionalPostRector/Source/CommentRemoverPostRector.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\Tests\PostRector\AdditionalPostRector\Source;
+
+use PhpParser\Node;
+use PhpParser\Node\Stmt;
+use Rector\PostRector\Rector\AbstractPostRector;
+
+final class CommentRemoverPostRector extends AbstractPostRector
```

Comment:
We add a sample post rector that removes all existing comments if the file has a namespace